website/src/categories/components/dialog/accessibility.md

```diff
@@ -7,32 +7,38 @@ eleventyNavigation:
 ---
 <section>
 
-## Dialog accessibility content
+## Keyboard interactions
 
-This is the accessibility content
+<div class="ds-table-wrapper">
 
-</section>
-
-<section>
+|Command|Description|
+|-|-|
+|Tab|Moves focus to the next tabbable element inside the dialog. If focus is on the last tabbable element, it wraps back to the first.|
+|Shift + Tab|Moves focus to the previous tabbable element inside the dialog. If focus is on the first, it wraps to the last.|
+|Escape (ESC)|Closes the dialog.|
```

Comment:
Shouldn't we mention that this behavior is disabled when `disableCancel` is set?